docs/advanced_examples/GLMComparison.ipynb

```diff
@@ -24,7 +24,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
Some notebooks haven't run normally, it's worth refreshing them with the CI action